### PR TITLE
Globebrowsing Fade-In/Out Fix and Assets Updates (Issue #1209)

### PR DESCRIPTION
**Pull request by GPayne**
Fixed fade out of globebrowsing labels so that fading-out labels when close to the surface works with fade-in as well.
Also updated all asset files that use the labels so that the fade in/out ranges, culling, and font sizes look good.

### CODE REVIEW

File: data/assets/scene/solarsystem/planets/earth/magnetosphere.asset [hunk 1]
```diff
@@ -0,0 +1,67 @@
+local assetHelper = asset.require('util/asset_helper')
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/magnetosphere.asset:1]: I assume this file made it in by accident? :D

File: modules/globebrowsing/src/globelabelscomponent.cpp [hunk 2]
```diff
@@ -50,7 +50,7 @@ namespace {
 
     constexpr const char* KeyLabelsFileName = "FileName";
 
-    constexpr const double LabelFadeRangeConst = 1500.0;
+    constexpr const double LabelFadeOutLimitAltitude_meters = 25000.0;
```
Comment (alexanderbock) [inline modules/globebrowsing/src/globelabelscomponent.cpp:53]: Would prefer: `LabelFadeOutLimitAltitudeMeters` to keep the same camelCase naming convention

**Review by alexanderbock — CHANGES_REQUESTED** [review]

File: data/assets/scene/solarsystem/planets/earth/magnetosphere.asset (hunk 1 above)
Comment (GPayne) [reply data/assets/scene/solarsystem/planets/earth/magnetosphere.asset]: Oops! Removed it.

File: modules/globebrowsing/src/globelabelscomponent.cpp (hunk 2 above)
Comment (GPayne) [reply modules/globebrowsing/src/globelabelscomponent.cpp]: Changed this.

**Comment by alexanderbock** [comment]
👍

**Pull request merged**